tweet_recommendations/datasets_access.py

```diff
@@ -0,0 +1,94 @@
+import os
+import pandas as pd
+import pickle
+from collections import Counter
+
+
+PATH = os.path.dirname(os.path.abspath(__file__)) + '/../'
```

Comment:
Jak chcemy być pro cross platform, to możemy wszędzie używać `os.path.join`, żeby łączenie ścieżek prawidłowe. O ile takie łączenie działa na windowsie, tak np. windowsowe `\\path\\to\\something` nie działa na linuxie. Polecam też `from pathlib import Path`, ale to na przyszłość :)